docs/charts/doughnut.md

```diff
@@ -62,6 +62,8 @@ The doughnut/pie chart allows a number of properties to be specified for each da
 | `hoverBackgroundColor` | `Color[]` | The fill colour of the arcs when hovered.
 | `hoverBorderColor` | `Color[]` | The stroke colour of the arcs when hovered.
 | `hoverBorderWidth` | `Number[]` | The stroke width of the arcs when hovered.
+| `legendSymbol` | `String[]` | Symbol to display as label when usePointStyle is false (size is based on fontSize except for large symbol (ending with Lg) where boxWidth is used in this case). Values are rectLg, lineLg, rectRoundedLg and other values from [PointStyle](./elements.md/#point-styles)
```

Comment:
I think it might make sense to split large vs small into a separate boolean.

Reply:
or, if boxWidth is defined, use it else use the fontSize

Reply:
I think large and small doesn't make sense for all symbol types. Most of them should have the width = height which is based on fontSize. That's why I created specific type for the 3 symbols in the list that in my opnion could be large (line, rect and rectRounded). I can create a boolean, but this won't impact symbol that should keep their aspect ratio (width=height).
Same comment for option2...for symbol that should keep their aspect-ratio using boxWidth by default is probably not good.

Reply:
Hi,
all the current symbols coming from pointStyle are based on Font Size. From those symbols , in my options, only should be displayed as large width. Rect (only available symbol right now if we don't use point style), line and rectRounded.
I can add a boolean to identify when a user wants to display large symbol, but I think it should only affect those 3 symbol types. Since it is only for 3 symbol I was proposing to add them with Lg at the end, but could be better to add a Boolean that impact only minority of symbol since it could impact more in the future.... Let me know what your preference is.